index.html

```diff
@@ -14,31 +14,23 @@
   </head>
 
   <body>
+    <section>
     <!--header-->
     <header class="header">
-<<<<<<< HEAD
-      <img class="logo" src="./logo.svg" alt="LOGO" />
+      
+      <div class="logo">
+        <a href="index.html"><img  src="./logo.svg" alt="LOGO" /></a>
+      </div>
 
       <nav>
         <a class="dropdown" href="">
           <i class="fa fa-bars" aria-hidden="true"> </i
         ></a>
```

Comment:
- [ ] [Optional] [Highly recommended] Please add your hamburger button inside a `<button>` tag instead of link `<a>` tag for semantic reasons.